charts/px-central/templates/px-backup/pxcentral-backup.yaml

```diff
@@ -11,6 +11,36 @@ metadata:
     app.kubernetes.io/component: px-backup
 {{- include "px-central.labels" . | nindent 4 }}
 ---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: px-backup-cluster-role
```

Comment:
can we remove cluster-role from the name itself?

Reply:
Role def, previously, it was like px-backup-role-binding and px-backup-role. So I followed the same